language-server/modules/langserver-compiler/src/main/java/org/ballerinalang/langserver/compiler/config/DataMapperConfig.java

```diff
@@ -0,0 +1,24 @@
+package org.ballerinalang.langserver.compiler.config;
+
+/**
+ * Ballerina CodeLens Configuration.
+ */
+public class DataMapperConfig {
+    private final boolean enabled;
+    private final String url;
+
+    DataMapperConfig() {
+        this.enabled = false;
+        this.url = "";
+    }
+
+    public boolean isEnabled() {
+
+        return enabled;
+    }
+
+    public String getUrl() {
+
```

Comment:
Remove this extra line feed, check for other places as well

Reply:
Done